include/libswiftnav/signal.h

```diff
@@ -37,16 +58,14 @@ enum band {
 
 typedef struct __attribute__((packed)) {
   u16 sat;
-  u8 band;
-  u8 constellation;
+  enum code code;
+  u8 reserved;
```

Comment:
Agree. We have different structs and converter functions for internal and external (SBP message) representations. Internal representation shouldn't matter.